src/dht.cpp

```diff
@@ -2563,8 +2567,15 @@ Dht::storageRefresh(const InfoHash& id, Value::Id vid)
         }
 
         auto expiration = s->second.refresh(now, vid, types);
```

Comment:
little comment but for clarity we can use "auto [storage, expiration]" and avoid a lot of .first after this

Reply:
OpenDHT is still C++14